content/_releases/22/22.md

```diff
@@ -0,0 +1,235 @@
+
+
+### Re-authentication for sensitive actions (continuation)
+
+This is the continuation of our effort to prevent accidental change or deletion
+of sensitive information. The first part was completed during the
+[22.20](/releases/22/20#re-authentication-for-sensitive-actions) platform release.
+
+This time we extend the list of actions for which you will be asked to enter your
+password to authenticate:
+
+*   Change user roles in Contracts and Workspaces.
+*   Add or remove SSH Keys.
+*   Remove member from Contract.
+
+> **Again, please note**: You will be asked to re-authenticate once every 6 hours.
+
+
+{% if site.data.tenant.name == "elastic.io" %}
+
+## HELM3
+
+We constantly improve different aspects of HELM3 deployments in collaboration
+with our partners. This section lists updates and improvements included in this release.
+
+
+### Enforcing Maester object size
+
+We introduce a new variable (`MAESTER_MAX_SIZE_PER_OBJECT` in bytes) you can use to control
+the maximum object size accepted by the Maester service. The default maximum value
+is set 1GB. The recommended range is from 100MB to 1GB.
+
+
+## OEM Related
+
+Information in this section is intended for our customers who use
+[OEM version of the elastic.io platform](https://www.elastic.io/saas-embedded-integration/).
+
+### Email domain blacklist
+
+If your tenant has an an open registration page then you can block registration
+from any domain by adding it into the domain blacklist. For this to work in your
+tenant, make an [API call]({{site.data.tenant.apiDocsUri}}/v2/#update-a-tenant)
+to update your tenant records using `tenantAdmin` privileges:
+
+*   Enable the registration page - set the `attributes.hide_register` parameter to `false`.
+*   Add records to the blacklist - create the `attributes.email_domains_blacklist` record to contain comma separated domain names in an array object like `["example.com", "foo.bar"]`.
+
+### Email verification with regular signup page
+
+During the 20.20 platform release we introduced [email verification process](/releases/22/20#email-verification-during-registration) which worked only when the [`signin_v_2` were used](/releases/22/20#setup-email-verification).
+
+Now you can use the email verification process even if you are using a regular
+signup page in your tenant. To enable this feature in your tenant the following
+steps must be done using an [API call]({{site.data.tenant.apiDocsUri}}/v2/#update-a-tenant)
+to update your tenant records using `tenantAdmin` privileges:
+
+*   Enable the registration page - set the `attributes.hide_register` parameter to `false`.
+*   Enable `email_verification` feature flag - set `attributes.feature_flags.email_verification` to `true`.
+
+{% endif %}
+
+## Fixed bugs
+
+*   Fixed the user interface bug when the Execution page would not load for some time when the workspace contained execution of a flow who's author deleted his/her account permanently. Now you will see `[Deleted User]` instead.
```

Comment:
Slow loading of the executions page for workspaces containing executions belonging to a deleted user's flows.